[BUG] Module not found: Error: Can't resolve 'attr-accept' in '/path'

**Describe the bug**
I am getting this error when I try to install react-dropzone.

Module not found: Error: Can't resolve 'attr-accept' in '/path'

**To Reproduce**
1. try to install react-dropzone.

**Expected behavior**
This was working prior to the latest version release.


**Desktop (please complete the following information):**
 - macOs Sequioa

**Additional context**
Add any other context about the problem here.

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: "20.x"
+          node-version: "14.x"
       - run: npm ci
       - run: npx semantic-release
         env:

